lib/IRGen/GenType.cpp

```diff
@@ -423,6 +423,19 @@ static llvm::Value *computeExtraTagBytes(IRGenFunction &IGF, IRBuilder &Builder,
   return phi;
 }
 
+static Address alignAddress(IRGenFunction &IGF,
+		            Address originAddress,
+			    llvm::Value *shift){
+  auto &Builder = IGF.Builder;
+  auto &IGM = IGF.IGM;
+  auto *alignedAddress = Builder.CreateBitCast(originAddress, IGM.Int8PtrTy).getAddress();
+#if defined(__linux__) && defined(__BIG_ENDIAN__)
```

Comment:
Same.

Reply:
why use ``__linux__`` here and ``__LP64__ `` in the other change above?

Reply:
For `GenType` change, we have only verified on our linux environment since that's the only environment available to us. The other change is bit size sensitive, the change only applies for 64 bit platforms.